fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -936,10 +912,10 @@ protected Response createUpdateResponse(final FedoraResource resource, final boo
         }
     }
 
-    protected static MediaType getSimpleContentType(final MediaType requestContentType) {
+    protected static String getSimpleContentType(final MediaType requestContentType) {
```

Comment:
MediaType is an HTTP construct and in many places we to `mediaType.toString()`. It seemed easier to just deal in strings unless we really needed to know that the type was `image` and the subType was `jpeg`.

Because left over code that expected MediaType I casted to try to keep those places running but they could probably be removed.

Reply:
Okay, I didn't think there were any other calls to this method other than the ones that go through MediaType.valueOf, but maybe more are expected?

Reply:
Once this PR is merged, I can have a look and see if this function is even needed anymore.